scripts/libmetal.mk

```diff
@@ -80,6 +84,7 @@ metal-bsp:\
 	   $(BSP_DIR)/metal.default.lds \
 	   $(BSP_DIR)/metal.ramrodata.lds \
 	   $(BSP_DIR)/metal.scratchpad.lds \
+	   $(BSP_DIR)/metal.privilege.lds \
```

Comment:
where is privilege.lds come from? you meant freertos?

Reply:
Yes, that's it, i miss this one.